go.mod

```diff
@@ -19,8 +20,11 @@ require (
 	go.uber.org/multierr v1.6.0 // indirect
 	go.uber.org/zap v1.16.0
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
-	k8s.io/api v0.19.2
-	k8s.io/apimachinery v0.19.2
-	k8s.io/client-go v0.19.2
-	sigs.k8s.io/controller-runtime v0.7.0
+	k8s.io/api v0.20.2
+	k8s.io/apimachinery v0.20.2
+	k8s.io/client-go v0.20.2
+	k8s.io/utils v0.0.0-20210111153108-fddb29f9d009
+	sigs.k8s.io/controller-runtime v0.8.3
 )
+
+replace github.com/RHEcosystemAppEng/dbaas-operator v1.0.0 => github.com/RHEcosystemAppEng/dbaas-operator v1.0.1-0.20210616190557-f8ab366a6505
```

Comment:
Why do a replace here instead of changing the version at line 6? Are you running off a local dbaas-operator copy that you're replacing to?

Reply:
This is the data structures of dbaas-operator are still subject to changes.  In this way we can simply pick up the latest changes from the repo. Once we have finalized dbaas-operator we can use a predefined version here.